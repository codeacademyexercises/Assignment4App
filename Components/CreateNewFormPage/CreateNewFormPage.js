import React, { Component } from 'react';
import {
  Text, View,
} from 'react-native';
import styles from './CreateNewFormPage.style';

export default class CreateNewFormPage extends Component {
  render() {
    return (
      <View>
        <Text>
                    hi!
        </Text>
      </View>
    );
  }
}
