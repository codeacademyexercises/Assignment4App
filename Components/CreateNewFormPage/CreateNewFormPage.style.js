import {
  StyleSheet,
} from 'react-native';

export default StyleSheet.create({
  FormCreateContainer: {
    marginTop: 50,
    justifyContent: 'space-around',
    alignItems: 'stretch',
  },
  FormNameInput: {
    marginLeft: 70,
    height: 40,
    width: 250,
    fontSize: 30,
    borderBottomWidth: 1,
    borderColor: 'black',
  },
  createFieldButton: {
    backgroundColor: '#4154AF',
    marginTop: 30,
    marginRight: 30,
    alignSelf: 'flex-end',
    justifyContent: 'center',
    alignItems: 'center',
  },
  createFieldButtonText: {
    margin: 4,
    fontSize: 16,
    color: 'white',
  },
  SaveFormButton: {
    backgroundColor: '#4154AF',
    justifyContent: 'center',
    alignItems: 'center',
  },
  FieldsScrollView: {
    height: 800,
  },
});
