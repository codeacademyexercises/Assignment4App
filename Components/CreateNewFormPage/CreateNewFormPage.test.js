import React from 'react';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CreateNewFormPage from './CreateNewFormPage';

configure({ adapter: new Adapter() });

describe('<Header /> ', () => {
  it('snapshot test', () => {
    const tree = renderer.create(<CreateNewFormPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
