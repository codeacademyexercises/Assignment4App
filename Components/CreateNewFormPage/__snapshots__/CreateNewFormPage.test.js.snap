// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header />  snapshot test 1`] = `
<View>
  <Text>
    hi!
  </Text>
</View>
`;
